docs/desktop-integration.rst

```diff
@@ -0,0 +1,86 @@
+Desktop Integration Guide
+=========================
+
+:doc:`conventions` covers the essential aspects of Linux desktop integration. This page provides further information on optional desktop integration features. It also provides guidance on how applications can ensure that their user interfaces fit into the whole range of Linux desktops and distributions.
+
+This information is primarily intended for developers who are new to Linux, however it is also relevant to desktop-specific Linux applications who wish to target a broader range of Linux environments.
+
+While targeting the Linux desktop ecosystem might seem challenging, the existence of common standards, in combination with these simple guidelines, means that supporting the full range of Linux environments needn't be difficult. This guide aims to show that Linux is an accessible platform for existing applications.
+
+Standardized integration options
+--------------------------------
+
+The following are standardized integration options, which can be accessed on any Linux desktop or environment.
+
+Locale detection
+````````````````
+
+Application toolkits, such as Electron, GTK and Qt, provide built-in support for detecting which locale to use. Otherwise, the ``setlocale`` function can be used.
+
+Portals
+```````
+
+Portals are the framework for securely accessing resources from outside an application sandbox. They provide a range of common features to applications, including:
+
+- Determining network status
+- Opening a file with a file chooser
+- Opening URIs
+- Preventing the device from suspend/sleep/powering off
+- Printing
+- Sending email
+- Taking screenshots and screencasts
+
+Toolkits like GTK and Qt provide transparent support for portals. If you are not using one of these toolkits, it is possible to access the portals API directly. See the `portals API documentation <https://flatpak.github.io/xdg-desktop-portal/portal-docs.html>`_ for more information.
+
+Notifications
+`````````````
+
+Applications can show system notifications on any distribution or desktop by using `libnotify <https://developer.gnome.org/libnotify/>`_. GTK applications can also use the `GNotification <https://developer.gnome.org/gio/stable/GNotification.html>`_ API. Electron transparently supports Linux desktop notifications.
```

Comment:
`libnotify` does not and will never use the portals.

Reply:
I added a note that recommends portals over libnotify.